### ISSUE TITLE
Make model trained on brace available

### ISSUE THREAD

**Issue by jaderxnet** [dmoltisanti/brace#3]
I saw in the article that you already trained a pre-trained HRNet model on Brace. However, the training code was not available. Could you make the code and/or model available so I can test its performance in other dances?

**Comment by dmoltisanti** [comment]
Hey, thanks for your interest! Unfortunately I was not in charge of this for the paper, hopefully Jinyi can help with this! @Neoutlier

**Comment by DA-NDI** [comment]
Thanks for the dataset! We are also using your dataset for our ml project at university (credits will be included :)

 I have the same question, any code for training of your model? Will be interesting to see your implementation

**Comment by dmoltisanti** [comment]
Hi, I will ping the other author to see if he can help with this, thanks!

**Comment by Neoutlier** [comment]
Sorry I've only seen the request now. Sadly I currently do not have the code or checkpoint from the project. However for the pose estimation training you can use [mmpose](https://github.com/open-mmlab/mmpose) and finetune with the annotated keypoints.

**Comment by jaderxnet** [comment]
> Thanks for the dataset! We are also using your dataset for our ml project at university (credits will be included :)
> 
> I have the same question, any code for training of your model? Will be interesting to see your implementation

Hello DA-NDI, I believe have a litle advanced to reply, but not with good results yet, i think now i need change the weights. I share here my best notebook try reproduce the brace training. Please collaborate with me, maybe together we can achieve. 

[Shared Folder with train and test data made by me](https://drive.google.com/drive/folders/1s2-hC6nySQVGHe14Ov9NCGVrxLn1XuZT?usp=sharing)
[Shared Colab Notebook](https://colab.research.google.com/drive/1X9X2N7cFwYZ-h94pytkqCtV83BcC8rea#scrollTo=1jgcC23Dlm6b)
[Shared folder results](https://drive.google.com/drive/folders/1JrwMh0JkUGBQw7L4G1OY9EVT4KoCXu2J?usp=sharing)

**Comment by DA-NDI** [comment]
Thanks! Will look into that. I started doing it during presentation of ml projects :) Your dataset helped us a lot

**Comment by DA-NDI** [comment]
We will share our results later. Comparison table:
<img width="654" alt="Screen Shot 2024-05-30 at 11 33 53" src="https://github.com/dmoltisanti/brace/assets/16738255/0bb99226-c50e-40e3-b194-f8957c38db6f">
I struggled with lstm (got 30% validation accuracy and huge overfitting but my teammate got pretty ok results).
